client/app/telecom/telephony/alias/configuration/stats/easyHunting/telecom-telephony-alias-configuration-stats-easyHunting.html

```diff
@@ -0,0 +1,217 @@
+            <dt data-translate="telephony_alias_configuration_stats_calls_answered"></dt>
+            <dd data-ng-bind="StatsEasyHuntingCtrl.stats.callsAnswered"></dd>
+            <dt data-translate="telephony_alias_configuration_stats_calls_lost"></dt>
+            <dd data-ng-bind="StatsEasyHuntingCtrl.stats.callsLost"></dd>
+            <dt data-translate="telephony_alias_configuration_stats_calls_wait_average"></dt>
+            <dd data-ng-bind="StatsEasyHuntingCtrl.getAverageWaitTime() | duration"></dd>
+            <dt data-translate="telephony_alias_configuration_stats_calls_duration_average"></dt>
+            <dd data-ng-bind="StatsEasyHuntingCtrl.getAverageCallTime() | duration"></dd>
+        </dl>
+    </div><!-- stats -->
+
+    <!-- CALLS -->
+    <div class="telecom-section-content" data-ng-if="!StatsEasyHuntingCtrl.isLoading">
+        <div class="widget-presentation">
+            <header class="widget-presentation-header">
+                <h2 class="widget-presentation-title" data-translate="telephony_alias_configuration_mode_calls_title"></h2>
+            </header>
+        </div>
+        <dl>
+            <dt data-translate="telephony_alias_configuration_mode_calls_ongoing"></dt>
+            <dd data-ng-bind="StatsEasyHuntingCtrl.getOngoingCalls().length"></dd>
+            <dt data-translate="telephony_alias_configuration_mode_calls_pending"></dt>
+            <dd data-ng-bind="StatsEasyHuntingCtrl.getPendingCalls().length"></dd>
+            <dt data-translate="telephony_alias_configuration_mode_calls_maxwait"></dt>
+            <dd>
+                <elapsed-time data-ng-if="StatsEasyHuntingCtrl.getMaxWaitTime()"
+                              data-from="StatsEasyHuntingCtrl.getMaxWaitTime()"></elapsed-time>
+                <span data-ng-if="!StatsEasyHuntingCtrl.getMaxWaitTime()">-</span>
+            </dd>
+        </dl>
+        <div class="table-responsive-vertical">
+            <table class="table table-pretty table-hover table-striped">
+                <thead>
+                    <tr>
+                        <th scope="col" data-translate="telephony_alias_configuration_mode_calls_number"></th>
+                        <th scope="col" data-translate="telephony_alias_configuration_mode_calls_queue_time"></th>
+                        <th scope="col" data-translate="telephony_alias_configuration_mode_calls_time"></th>
+                        <th scope="col" data-translate="telephony_alias_configuration_mode_calls_agent"></th>
+                        <th scope="col" data-translate="telephony_alias_configuration_mode_calls_action"></th>
+                    </tr>
+                </thead>
+                <tbody data-ng-if="!StatsEasyHuntingCtrl.isLoading">
+                    <tr data-ng-repeat="call in StatsEasyHuntingCtrl.callsPaginated track by call.id">
+                        <td data-title="{{:: 'telephony_alias_configuration_mode_calls_number' | translate }}">
```

Comment:
For the `callerIdNumber` you can replace your `<td>` with a `<th scope="row">` nope?